ko/pl.rst

```diff
@@ -0,0 +1,1383 @@
+
+:meta-keywords: cubrid sql, pl/cubsql
+:meta-description: This chapter describes PL/CUBSQL Spec.
+
+*****************************
+Overview
+*****************************
+
+Stored Procedure 생성
+======================
+
+PL/CUBSQL은 Stored Procedure, 즉 프로시저와 함수를 생성하는데 사용된다.
+다음 문법을 따르는 CREATE PROCEDURE 문과 CREATE FUNCTION 문의 AS (또는 IS) 키워드 뒤에 PL/CUBSQL 코드를 써서
+현재 생성하고 있는 Stored Procedure의 동작을 기술한다.
+::
+
+    <create_procedure> ::=
+        CREATE [ OR REPLACE ] PROCEDURE <identifier> [ ( <seq_of_parameters> ) ]
+        { IS | AS } [ <seq_of_declare_specs> ] <body> ;
+    <create_function> ::=
+        CREATE [ OR REPLACE ] FUNCTION <identifier> [ ( <seq_of_parameters> ) ] RETURN <type_spec>
+        { IS | AS } [ <seq_of_declare_specs> ] <body> ;
+
+위 문법에서 프로시저나 함수의 *body*\는 PL/CUBSQL 실행문들을 포함하고
+그 앞의 선언부 *seq_of_declare_specs*\는 실행문들 안에서 사용될 변수, 상수, Exception 등을 선언한다.
+문법 요소들에 대한 자세한 내용은 :ref:`선언문 <decl>`\과 :ref:`실행문 <stmt>` 절을 참고한다.
+
+다음은 PL/CUBSQL을 사용해서 작성한 프로시저와 함수의 예이다.
+
+.. code-block:: sql
+
+    CREATE PROCEDURE insert_athlete(
+        a_name VARCHAR,
+        a_gender VARCHAR,
+        a_nation_code VARCHAR,
+        a_event VARCHAR) AS
+
+        PRAGMA AUTONOMOUS_TRANSACTION;
```

Comment:
It is trivial.

How about below for visibility?
```suggestion
    CREATE PROCEDURE insert_athlete(
        a_name VARCHAR,
        a_gender VARCHAR,
        a_nation_code VARCHAR,
        a_event VARCHAR)
AS
        PRAGMA AUTONOMOUS_TRANSACTION;
```

Reply:
OK. That looks better. Thank you